### ISSUE TITLE
Allow higher compression 

### ISSUE THREAD

**Issue by Magissia** [miku-nyan/Overchan-Android#80]
Hello, it would be  useful to be  able to save as compressed archive rather than just zip

**Comment by miku-nyan** [comment]
I do not think it will help a lot. Media files (images/jpeg/png/videos, mostly) are pretty compressed already and further compression will have little effect.
